packages/component/src/Styles/StyleSet/ActivityButton.ts

```diff
@@ -0,0 +1,45 @@
+export default function createActivityButtonStyle() {
+  return {
+    '&.webchat__activity-button': {
+      alignItems: 'center',
+      appearance: 'none',
+      background: '#fff',
+      borderRadius: 4,
+      border: '1px solid #d1d1d1',
+      color: '#242424',
+      display: 'flex',
+      gap: '4px',
+      justifyContent: 'center',
+      padding: '5px 12px',
+
+      '&:hover': {
+        background: '#f5f5f5',
+        border: '1px solid #c7c7c7',
+        color: '#242424'
+      },
+
+      '&:active': {
+        background: '#e0e0e0',
+        border: '1px solid #b3b3b3',
+        color: '#242424'
+      },
+
+      '&:focus-visible': {
+        background: '#fff',
+        outline: '2px solid #000',
+        outlineOffset: -2
+      },
+
+      '&:disabled': {
+        background: '#f0f0f0',
+        border: '1px solid #e0e0e0',
+        color: '#bdbdbd'
+      },
+
+      '& .webchat__activity-button__icon': {
+        height: 20,
+        width: 20
```

Comment:
```suggestion
        height: '20px',
        width: '20px'
```

I just think it is easier to reason about. E.g.:
```
  fontSize: 16,
  lineHeihgt: 1
```
vs
```
  fontSize: '16px',
  lineHeihgt: 1
```